hack/tools/create_kind.sh

```diff
@@ -97,7 +98,7 @@ case "$op" in
     cleanup)
         header_text "Uninstalling kind cluster"
         registry_delete || true
-        # certs_delete || true # Deactivated for now to simplify development when recreating a kind cluster
+        certs_delete || true
```

Comment:
why did you reactivate? Not saying that you shouldn't, but needs to understand

Reply:
this caused for a silent failure (which was very difficult to track down), as the `hack/registry` file was not refreshed causing the mounting of our local kind-cluster registry to have missing CA files, which in turn trigger the mysterious x503 errors which came and went. No (which this change) the issue will be detected during kind cluster creation and will be easy to detect and solve (solved in my case by rebooting the docker service which was holding the `hack/registry` directory and not allowing it to be recreated correctly)

Reply:
Got it, thanks.